clarity_ext/context.py

```diff
@@ -124,6 +131,17 @@ def output_result_file_by_id(self, file_id):
     def output_result_files(self):
         return self.artifact_service.all_output_files()
 
+    @property
+    def user_initials(self):
```

Comment:
I think we should start thinking about limiting the properties we add to the context, as it will otherwise start getting rather crowded. So in this case, perhaps it would suffice that the extension writer could write `context.current_user.initials` instead of having a property for that.